core-activities.md

```diff
@@ -1,16 +1,16 @@
 # Core Activities
 
-## Implementation and Decommissioning
-We lead strategy for implementation and decommissioning of systems related to archival functions. We evaluate workflows, systems, and processes holistically, with an eye towards removing redundancy and eliminating unnecessary manual intervention so that our colleagues are better able to use their unique skills in acquiring, preserving, and providing access to archival collections.
+## Implement and Develop Systems
+We lead strategy for the selection, implementations, and decommissioning of systems key to archival functions. We evaluate workflows, systems, and processes holistically, with an eye towards removing redundancy and eliminating unnecessary manual intervention so that our colleagues are better able to use their unique skills in acquiring, preserving, and providing access to archival collections. We develop systems that fit the RAC's needs and use cases, as well as work with open source applications and their communities.
```

Comment:
```suggestion
We lead strategy for the selection, implementation, and decommissioning of systems key to archival functions. Evaluating workflows, systems, and processes holistically, we try to removing redundancy and eliminate unnecessary manual intervention so that our colleagues are better able to use their unique skills in acquiring, preserving, and providing access to archival collections. We develop systems that fit the RAC's needs and use cases, and also work with open source applications and their communities.
```